src/cbc_sdk/platform/devices.py

```diff
@@ -254,6 +255,46 @@ def get_vulnerabilties(self):
         """
         return VulnerabilityQuery(Vulnerability, self._cb, self)
 
+    @property
+    def nsx_available(self):
+        """
+        Returns whether NSX actions are available on this device.
+
+        Returns:
+            bool: True if NSX actions are available, False if not.
+        """
+        return self._info['deployment_type'] == 'WORKLOAD' and self._info['nsx_enabled']
+
+    def nsx_remediation(self, tag, set_tag=True):
+        """
+        Start an NSX Remediation job on this device to change the tag.
+
+        Required Permissions:
```

Comment:
I think that should be specified in a different way otherwise it looks like something (python related) is related to use the method. Maybe just a single sentence like `The required permissions for the API are appliances.nsx.remediation(EXECUTE)`.